assignments/array-methods.js

```diff
@@ -55,29 +55,82 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
-console.log(fullName);
+
+//  Study Group Solution
+let fullName = [[runners.first_name] + [runners.last_name]];
+ let result = fullName.map(function(runners) {
+     return [runners.first_name + runners.last_name];
+ })
+ console.log(fullName);
+
+
+
+
+// Q&A Solution
+
+ let fullName=[]
+     fullName = runners.item  {
+     fullName.push('${item.first_name}, $(item.last_name}')
+     return (fullName);
+ }
+
+ console.log(fullName)
+
+
+
+
+
+
+
 
 // ==== Challenge 2: Use .map() ====
-// The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
+// The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. 
+// Convert each first name into all caps and log the result
+
+//  Q&A way
+  let allCaps = runnners.map(item=>item.first_name.toUpperCase());
+  console.log(allCaps);
+
```

Comment:
The `map` method returns an array, of the the returns it gets from the callback you give it which is why this works.